### DISCUSSION TITLE
Kum saati sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#235] [Zeka Sorusu]
iki kum saatinden biri 4 dakikayı diğeri ise 7 dakikayı ölçüyor. bu kum saatleri ile 9 dakikalık zaman nasıl ölçülür?

**Comment by Mertcakm** [comment]
Dört ve yedi aynanda başlayacak dört bittiğinde onu ters çevireceksin sonra yedi bittiğinde dokuz dakkalık süreyi tutmaya başlayacaksın dördün kalan 1 dakikası olacak o bitince yeniden ters çevireceksin böylece 5 dakika tutmuş olucaksın ardından dördü yeniden çevirince 9 dakika etmiş olacak.

**Reply by sonsuzus** [reply]
Noktalama kullanmayıp algoritma şeklinde yazmayınca pek anlaşılmıyor.
4 ve 7 başladı.
4 Bitti ters çevirdik, 7 lik olan 4dk da.
4 ün 1 dakikası kaldı, 7 bitti
Şimdi ne yapıyoruz?

**Reply by Mertcakm** [reply]
4ün bir dakikası bitti
Dördü ters çeviriyoruz
Bitince yeniden ters çeviriyoruz
